### DISCUSSION TITLE
Is there something similar for Android TV?

### DISCUSSION THREAD

**Discussion by JonathanJumper** [MonitorControl/MonitorControl#1081] [Ideas]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Discussion

I was wondering if there is something like this for android tv boxes, 
I cannot control monitor volume/turn on with it

**Comment by waydabber** [comment]
I am not aware of anything like that. Most TVs can change their brightness and volume via IR remote only or some proprietary (sometimes open) network control stuff utilized in the TV's remote app. It might be possible to start implementing these in MonitorControl but that is certainly a different project than this one.

I'll convert this one into a discussion.

**Comment by waydabber** [comment]
This one is related to this: https://github.com/MonitorControl/MonitorControl/discussions/1082
